dev-python/pysmi/pysmi-0.3.4.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 2017-2020 Gentoo Authors
+# Copyright 2017-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{7,8} )
+PYTHON_COMPAT=( python3_{7..9} )
 inherit distutils-r1
 
 DESCRIPTION="Python Lex & Yacc"
```

Comment:
This may not be relevant here, but I'll say it anyway: whenever you see no tests + pypi, remember to check GitHub for tests. Often, upstreams don't include it in the PyPi bundle and we have no real strong obsession with PyPi in Gentoo (i.e. GitHub is fine).

Reply:
Sure. Note that not all packages here have tests enabled, I did it for some, if you want me to do it for all I'll run through them again

Reply:
Don't break your back over it, but do it if you can for packages where it's easy at least. In future, of course, try do it as part of the routine, but let's be clear -- the Python 3.8+ stuff is already a nice win. :)

Reply:
(The main reason to do it is just so that in future, people don't assume it has no tests and that stabilisation has some meaning to it. So, it's *probably* less cost for you to do it now, as you have familiarity with the packages, but also, we're grateful you're doing this.)

Reply:
Agreed, I go with the same approach, let's clean up as much as we can while at it. In general if it works with `distutils_enable_tests` out of the box I'll just do it, otherwise depending on my current mood :smile: